server/src/controllers/userController.ts

```diff
@@ -35,12 +30,12 @@ export class UserController {
       const user = await UserService.getUser(id);
 
       if (!user) {
-        return res.status(404).send("User not found");
+        return res.status(StatusCode.NOT_FOUND).send("User not found");
       }
 
-      res.status(200).send(user);
+      res.status(StatusCode.OK).send(user);
     } catch (err: any) {
-      res.status(500).send({message:err});
+      res.status(StatusCode.NOT_FOUND).send({message:err});
```

Comment:
make sure when switching the status code to switch it to the same one. 

For example 500 is StatusCode.INTERNAL_SERVER_ERROR.

Now like i said it aint a big deal but if someone is expecting a code 500 and gets 404 then they will deal with it differently. So make sure you don't make changes like these without consulting the people who receive these responses.